ecs-cli/modules/compose/cli/ecs/app/service_command.go

```diff
@@ -136,3 +136,35 @@ func deploymentConfigFlags(specifyDefaults bool) []cli.Flag {
 		},
 	}
 }
+
+func loadBalancerFlags() []cli.Flag {
+	targetGroupArnUsageString := "[Optional] The full Amazon Resource Name (ARN) of the Elastic Load Balancing target group associated with a service."
```

Comment:
I am working on reviewing the text with others, and will update you once I have the final version.